docs/writing-docs/doc-block-canvas.md

```diff
@@ -6,9 +6,9 @@ Storybook's `Canvas` Doc Block is a wrapper featuring a toolbar that allows you
 
 ![Docs block with a story preview](./docblock-preview.png)
 
-## Working with the DocsPage
+## Working with Docs
```

Comment:
I think these snippets refer to how the automatically generated docs works. Are we moving away from calling that "DocsPage" (I'm cool with that) -- if so we still probably need to differentiate it from docs in general though (like these doc blocks only ever work in "Docs" so what does "Working with Docs" mean?)

The tag is still called `docsPage` although we could change this. Personally I've been using "autodocs" in my head a bit recently. Maybe we should call it "Automatic Docs" in our documentation?

So this section would be something like "Canvas in Automatic Docs"? WDYT @shilman?

Reply:
@tmeasday I'm actually been thinking the same about the naming. Autodocs has been going back and forward in my head as well. If @shilman is on board with it as well I'm more than glad in making the change.

Reply:
I think if we want to go with that name we should change the name of the option and tag also.

Reply:
I'm pretty neutral on the change. Fine with it if that's the consensus. I agree with @tmeasday that we should make it consistent if we do

Reply:
I think we need to be careful here about replacing "DocsPage" with just "Docs". In my mental model, both "DocsPage" and MDX fall under the umbrella of "Docs".

I like the suggestion of replacing "DocsPage" with "Automatic Docs" (`autoDocs` in config). Especially because this differentiates it from `parameters.docs.page`.